addon/i3dio/ui/object.py

```diff
@@ -150,9 +146,48 @@ class I3DNodeObjectAttributes(bpy.types.PropertyGroup):
     )
 
     collision_mask: StringProperty(
-        name="Collision Mask",
-        description="The objects collision mask as a hexadecimal value",
-        default=i3d_map['collision_mask']['default']
+        name="Collision Mask (deprecated)",

```

Comment:
Is there a way to map from the old way to the new way? Otherwise, if the conversion is supposed to be entirely manual, I think that we should just get rid of the property.

Reply:
I first thought that removing the prop from the property group would completely remove it from the dict, but after some more testing I see that the value is still kept in the dict. I added step to delete the prop inside the automatic (handler) conversion function now (convert_old_collision_masks)